packages/otel/src/bootstrap/main.ts

```diff
@@ -0,0 +1,65 @@
+import { GET_TRACER, SHUTDOWN_TRACERS } from '../constants.js'
+
+export const createTracerProvider = async (options: {
+  headers: Headers,
+  serviceName: string,
+  serviceVersion: string,
+  deploymentEnvironment: string,
+  siteUrl: string,
+  siteId: string,
+  siteName: string,
+}) => {
+  if (options.headers.has('x-nf-enable-tracing')) {
```

Comment:
nit: I would flip this and early-return here, to avoid having everything nested under this statement.

Reply:
Done 👍